scripts/hudson/quickstart.sh

```diff
@@ -156,7 +156,49 @@ function build_apache-karaf {
     exit -1
   fi
 }
+function clone_as {
+  REPO="https://github.com/wildfly/wildfly.git"
+  echo "Cloning AS sources from $REPO"
 
+  cd $WORKSPACE
+
+  if [ -d jboss-as ]; then
+    rm -rf jboss-as # start afresh
+  fi
+
+  git clone $REPO jboss-as || fatal "git clone $REPO failed"
+
+  if [ $REDUCE_SPACE = 1 ]; then
+    echo "Deleting git dir to reduce disk usage"
+    rm -rf .git
+  fi
+
+  cd $WORKSPACE
+}
+function build_as {
+  AS_BRANCH=27.0.0.Alpha1
+  echo "Building WildFly $AS_BRANCH"
+
+  cd $WORKSPACE/jboss-as
+
+  git checkout $AS_BRANCH || fatal "git checkout $AS_BRANCH failed"
+
+  # building WildFly
+  [ "$_jdk" -lt 17 ] && export MAVEN_OPTS="-XX:MaxMetaspaceSize=512m -XX:+UseConcMarkSweepGC $MAVEN_OPTS"
```

Comment:
Where does `_jdk` get initialised.

Reply:
Thanks @mmusgrov . I hope my last changes should fix that.